client/src/components/Brand.js

```diff
@@ -44,14 +41,8 @@ function Brand (props) {
     <MuiThemeProvider theme={theme}>
       <span>
         <Toolbar disableGutters>
-          <ButtonBase className={classes.root} component={Link} to='/dashboard'>
-            <DeviceHub
-              className={classNames(classes.leftIcon, classes.brandHover)}
-              style={{
-                width: 36,
-                height: 36
-              }} />
-            <Typography type='title'>eviteHub</Typography>
+          <ButtonBase className={classes.root} component={Link} to='/'>
+            <img src='/static/images/evitehub-logo.png' alt='EviteHub' />
```

Comment:
I like your logo! Is it a custom icon? 
Instead of replacing both icon and text, I would prefer to just replace the icon and keep the text in <Typography>. 
Please upload the logo image (just the icon) in higher resolution.. around 350x350